lib/src/super_extensions_base.dart

```diff
@@ -83,3 +83,13 @@ extension StringExtension on String {
           .map((str) => str.capitaliseFirstChar())
           .join(' ');
 }
+
+extension NumberExtension on num {
+  /// creates a Empty [SizedBox] with height as the given number
+  /// 
+  SizedBox get hSizedBox => SizedBox(height: toDouble());
+
+  /// creates a Empty [SizedBox] with width as the given number
```

Comment:
here it should be `Creates an empty SizedBox with [width] as the given number.
same for height.

Reply:
sure, updated